corehq/apps/reports/standard/deployments.py

```diff
@@ -194,9 +194,15 @@ def user_query(self, pagination=True):
                           .size(self.pagination.count)
                           .start(self.pagination.start))
         if self.selected_app_id:
+            # selecting reporting_metadata.last_sudmission as filter_path as this has the mapping
```

Comment:
Nit: It's good idea to add notes, but good to frame them around the context of the code rather than around bugs. So, might be good to reframe on the lines of 'Use reporting_metadata.last_submission to filter users by selected_app_id' or something else.